.ci/gitlab/test.yml

```diff
@@ -128,25 +128,25 @@ build-clash-dev:
 suite:vhdl:
   extends: .test-cache-local
   script:
-    - ./dist-newstyle/build/*/*/clash-testsuite-*/x/clash-testsuite/build/clash-testsuite/clash-testsuite -j$THREADS -p .VHDL --hide-successes --no-vivado
+    - $(.ci/find_cabal_bin.sh clash-testsuite clash-testsuite) -j${THREADS} -p .VHDL --hide-successes --no-vivado
 
 suite:verilog:
   extends: .test-cache-local
   script:
-    - ./dist-newstyle/build/*/*/clash-testsuite-*/x/clash-testsuite/build/clash-testsuite/clash-testsuite -j$THREADS -p .Verilog --hide-successes --no-vivado
+    - $(.ci/find_cabal_bin.sh clash-testsuite clash-testsuite) -j${THREADS} -p .Verilog --hide-successes --no-vivado
 
 suite:systemverilog:
   extends: .test-cache-local
   script:
-    - ./dist-newstyle/build/*/*/clash-testsuite-*/x/clash-testsuite/build/clash-testsuite/clash-testsuite -j$THREADS -p .SystemVerilog --hide-successes --no-modelsim --no-vivado
```

Comment:
Do we have modelsim on the docker images?

Reply:
Not in the docker, but the GitLab runners mount `/opt/tools` (or whatever it is called), so they can access modelsim.

Reply:
Still, this wasn't my intention :thinking: